provider/builtin/provider.go

```diff
@@ -169,15 +169,31 @@ func (p *builtinProvider) Evaluate(cap string, conditionInfo []byte) (lib.Provid
 			if err != nil {
 				return response, fmt.Errorf("Unable to find files using pattern `%s`: %v", pattern, err)
 			}
+		} else if len(cond.XML.Filepaths) == 1 {
```

Comment:
@pranavgaikwad @fabianvf 

I think we may want to make this documentable somehow, this feels bad with all the go -> yaml -> mustache -> yaml -> go. 

I don't have a solution but something to think about

Reply:
https://github.com/cbroglie/mustache/issues/83 

Seems there is a way to deal with this, but will still produce invalid JSON.